pkgs/top-level/all-packages.nix

```diff
@@ -20538,8 +20538,6 @@ with pkgs;
 
   dssi = callPackage ../development/libraries/dssi { };
 
-  duckdb = callPackage ../development/libraries/duckdb { };
```

Comment:
This gets picked up somewhere else I'm guessing?

Reply:
i moved the derivation to `pkgs/by-name/du/duckdb` and the `by-name` hierarchy doesn't require an entry in `all-packages.nix`. also, packages in `by-name` can use the auto-merge-bot for version updates triggered by `r-ryantm`.